### PR TITLE
ui: Topology Intentions Popovers

### PR DESCRIPTION
**Pull request by kaxcode**
- Creates the TopologyMetrics::Popover component
- Changes the grid-styling layout to not overlap and instead use negative margins
- Styling overrides for the ember-popover used

intentions_actions

### CODE REVIEW

File: ui/packages/consul-ui/app/components/topology-metrics/popover/index.scss [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  }
+}
+.deny-popover {
+  .body {
+    background-color: #FDFAFB;
+  }
+  h3 {
+    color: #5B1820;
+  }
+}
```
Comment (johncowen) [inline ui/packages/consul-ui/app/components/topology-metrics/popover/index.scss:87]: Are none of these colors in our variables color set?

**Review by johncowen — APPROVED** [review]
Seems ok to me, I think there are possibly some amends we can make here, like for example making use of our font/color variables etc. Kinda feels like we should get this in as is tho.

File: ui/packages/consul-ui/app/components/topology-metrics/popover/index.scss (hunk 1 above)
Comment (kaxcode) [reply ui/packages/consul-ui/app/components/topology-metrics/popover/index.scss:87]: Yes, replaced.

**Comment by johncowen** [comment]
This one missed a backport also -re-running

**Pull request merged**